app/src/main/java/com/dimagi/biometric/activities/BaseActivity.java

```diff
@@ -85,6 +93,7 @@ private void getIntentParams() {
             biometricType = BioCommon.BioType.Finger;
         }
         caseId = intent.getStringExtra(CASE_ID_PARAM);
+        templateStr = intent.getStringExtra(TEMPLATE_PARAM);
```

Comment:
The template is currently also retreived from `getRecordFromIntent()`. This function will be removed when the necessary Support Library changes have been done, and so the template will only be retrieved from `getIntentParams()`.